src/vivarium_census_prl_synth_pop/constants/metadata.py

```diff
@@ -275,3 +275,16 @@ class __Scenarios(NamedTuple):
 }
 
 SUPPORTED_EXTENSIONS = ["hdf", "parquet"]
+
+
+class DatasetNames:
+    """Container for Form names"""
```

Comment:
"Dataset" names

You could also import from pseudopeople instead?

Reply:
Oh, nm, it's kinda different w/ taxes.

Reply:
I decided that was out of scope for this.